documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -0,0 +1,84 @@
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-migration-plan-2-8_{context}"]
+= Creating a migration plan
+
+Use the {ocp} web console to create a migration plan. Specify the source provider, the virtual machines (VMs) you want to migrate, and other plan details.
+
+[WARNING]
+====
+Do not include virtual machines with guest-initiated storage connections, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts. These require either additional planning before migration or reconfiguration after migration.
+
+This prevents concurrent disk access to the storage the guest points to.
+====
+
+include::snip_plan-limits.adoc[]
+
+.Procedure
+
+. In the {ocp} web console, click *Plans for virtualization* and then click *Create Plan*.
++
+The *Create migration plan* wizard opens to the *Select source provider* interface.
+. Select the source provider of the VMs you want to migrate.
++
+The *Select virtual machines* interface opens.
+. Select the VMs you want to migrate and click *Next*.
++
+The *Create migration plan* pane opens. It displays the source provider's name and suggestions for a target provider and namespace, a network map, and a storage map.
+. Enter the *Plan name*.
+. To change the *Target provider*, the *Target namespace*, or elements of the *Network map* or the *Storage map*, select an item from the relevant list.
+. To add either a *Network map* or a *Storage map*, click the *+* sign anf add a mapping.
+. Click *Create migration plan*.
++
+{project-short} validates the migration plan, and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error.
++
+The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
++
+. Check the following items in the *Settings* section of the page:
+
+ifdef::vmware[]
+
+* *Warm migration*: By default, all migrations are cold migrations. For a warm migration, click the *Edit* icon and select *Warm migration*.
+* *Transfer Network*: The network used to transfer the VMs to {virt}, by default, this is the default transfer network of the provider. To edit the transfer network, click the *Edit* icon, choose a different transfer network from the list in the window that opens, and click *Save*. 
+* *Target namespace*: Destination namespace, by default, this is the {virt} cluster you chose as your destination provider. To edit the namespace, click the *Edit* icon, choose a different target namespace from the list in the window that opens, and click *Save*.  
```

Comment:
Regarding **Target namespace** setting:
This is the destination namespace to be used by all the migrated VMs.
The default value for this item is the current/active namespace.

WDYM by "this is the OpenShift Virtualization cluster you chose as your destination provider"?